app/styles/pages/events.scss

```diff
@@ -13,3 +13,24 @@
     margin-top: 2px;
   }
 }
+
+// Tickets styling
+
+.color.grey {
```

Comment:
Use the same set of colors from https://semantic-ui.com/elements/segment.html#colored for uniformity 

This will be moved to github.com/fossasia/open-event-theme in future.